app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
+            raise NotVaccinatedError("not vaccinated")
```

Comment:
Error messages should be more descriptive. Like: "Visitor must be vaccinated to enter the cafe"